bears/js/JSONFormatBear.py

```diff
@@ -90,9 +90,13 @@ def run(self, filename, file,
         diff = Diff.from_string_arrays(file, corrected)
 
         if len(diff) > 0:
+            out_mess = ('This file can be reformatted '
+                        'by following indentation.')
+            if json_sort:
+                out_mess = ('This file can be reformatted by sorting keys '
+                            'and following indentation.')
             yield Result(self,
-                         'This file can be reformatted by sorting keys and '
-                         'following indentation.',
+                         out_mess,
```

Comment:
Include a comment describing the 'out mess' parameter. Just an one line description is enough. I would also suggest to change the name of the variable to something more meaningful (maybe err_message). Rest of the code looks fine.

Reply:
`format_message` Will do I guess?